### ISSUE TITLE
are there some mistakes in kl_loss (line 282@AGNN_model.py)?

### ISSUE THREAD

**Issue by xiehbpku** [lylbaidu/AGNN#3]
In standard VAE,  it's item_var instead of item_z,
kl_loss = torch.mean(0.5 * torch.sum(torch.exp(item_z) + item_mu ** 2 - 1. - item_var, 1)) + \
                  torch.mean(0.5 * torch.sum(torch.exp(user_z) + user_mu ** 2 - 1. - user_var, 1))

"""
Hence we require the reconstructed embedding x′u to be similar with both the preference embed- ding (by the constraint) and the original attribute distribution (by the standard VAE). To summarize, the reconstruction loss function in our proposed eVAE is defined as follows.
***
where the first two terms are same as those in standard VAE, and the last one is our extension for the approximation part.
"""
